test-project-windows/.gitattributes

```diff
@@ -0,0 +1,86 @@
+﻿* text=auto
+
+# Unity files
```

Comment:
Do we need another test project? Ideally we'd reuse existing projects where possible.

We can merge with it but it's definitely something to keep in mind.

Reply:
Reason I made a copy of the project was because I needed to set the scripting backend to il2cpp. I'm not sure if we can do that from the cli since the target is just StandaloneWindows with no mention of the scripting backend. Open to suggestions on how to get it to test with the right backend without a separate project.

Reply:
That way I do it in my own project is I overwrite the `scriptingBackend` in the `ProjectSettings.asset` before running the build. But that also requires me to include `allowDirtyBuild: true`, so not sure if that makes sense here.

Reply:
Just for testing the integrity of the action it might be worth it.

We do have some examples of where we already do this. ([example 1](https://github.com/game-ci/docker/blob/c82b022be4c88573b2f4d5f4aece4945e0824b30/.github/workflows/test.yml#L208), [example 2](https://github.com/game-ci/docker/blob/c82b022be4c88573b2f4d5f4aece4945e0824b30/.github/workflows/test.yml#L230))

Reply:
Sounds good. I'll see about getting something similar implemented. Thanks for pointing me in the right direction 🙂